inc/class-bento-settings-controller.php

```
@@ -50,4 +55,136 @@ public function handle_fetch_authors(): void {
+     */
+    public function handle_verify_events_queue(): void {
+        check_ajax_referer('bento_settings', '_wpnonce');
+
+        if (!current_user_can('manage_options')) {
+            wp_send_json_error(['message' => 'Permission denied']);
+            return;
+        }
+
+        try {
+            // Remove the bento_events_queue option from the database
+            $result = delete_option('bento_events_queue');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (Unused Code Rules)

(UnusedLocalVariable)